pkg/authn/authn.go

```diff
@@ -14,6 +14,11 @@ type Session struct {
 	SuperAdmin   bool
 }
 
+func (s *Session) UpdateSession(DomainID string) {
```

Comment:
No need of this changes function, 

Instead check and invalidate authn session to empty object , if authn session domain id is not equal to domainID 

We could not have these logic for all endpoints, it is needed only required for domains, things, groups and channels 

For users this logic is no needed logic.